vrc-get-gui/components/providers.tsx

```diff
@@ -71,22 +78,14 @@ export function Providers({children}: { children: React.ReactNode }) {
 				pauseOnFocusLoss
 				draggable
 				pauseOnHover
-				theme="light"
+				theme="colored"
```

Comment:
Resolving this suggestion seems to be really tricky.

The first issue I'm encountering is I'm not able to override toastify's css variables in `globals.css` as toastify seems to override my configuration as shown in the following image. I can't think of a workaround for this problem.

![image](https://github.com/vrc-get/vrc-get/assets/54484875/fad937ec-736f-4347-b703-8b341ee57288)

I've tried to solve this by re-ordering the imports in `app/layout.tsx` like so, but I got the same result.

```typescript
import 'react-toastify/dist/ReactToastify.css';
import "./globals.css";
```

The next issue is a minor bug when the theme is changed through settings and the app isn't restarted, toastify's theme will not change theme appropriately as I'm not able to access the state in `settings.tsx`. This would more than likely require a store like [Redux](https://redux.js.org/) to fix unless there's another solution I'm not thinking of. I'd like to know if adding a store is something you deem appropriate for this situation!

One solution would be to outright migrate toastify over to shadcn's [toast](https://ui.shadcn.com/docs/components/toast) component but I assume this would be out of scope for this PR? Let me know!

Reply:
Thank you for research. I'm going to keep colored for now and I'll consider.